docs/rfcs/sync-manager.md

```diff
@@ -0,0 +1,171 @@
+# Sync manager
+
```

Comment:
nit: Probably worth making lines shorter so we can respond to individual points. 80 or 100 line columns are probably fine. There are good plugins for this (just hit alt+q with https://marketplace.visualstudio.com/items?itemName=stkb.rewrap or https://packagecontrol.io/packages/Wrap%20Plus).

Reply:
yeah, I never know whether .md should be explicitly wrapped or not. I'll re-wrap it.